sup.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+
+from brutal.core.plugin import cmd
+from datetime import datetime
+import admanbot.plugins.calendar_sk as calendar_sk
+
+
+@cmd
+def sup(event):
+    """Inform you about current day"""
+    today = datetime.today()
+
+    day = today.day
+    month = today.month
+
+    name = calendar_sk.NAMEDAY[month-1][day-1]
+
+    try:
+        hol = calendar_sk.holiday[str(month)][str(day)]
+    except KeyError:
+        hol = None
+
+    try:
+        intday = calendar_sk.INTERNATIONAL_DAY[str(month)][str(day)]
+    except KeyError:
+        intday = None
+
+    name_str = '' if name is None else ', Nameday: {0}'.format(name)
+    hol_str = '' if hol is None else ', {0}'.format(hol)
+    intday_str = '' if intday is None else ', {0}'.format(intday)
```

Comment:
Wouldn't it be better, instead of `try ... except ...` and these `if`s, to use the `dict.get` method, which takes (optional) second parameter, value to be returned if no such key exists? Also, why not put empty strings to nameday calendar instead of `None`s?